### ISSUE TITLE
You took our $10 and never fixed anything

### ISSUE THREAD

**Issue by andveg38** [pandasa123/Things3CalendarSyncIssues#6]
You have been missing in action ever since problems were reported by those that paid you $10 for your app. Please update us.

**Comment by blacklevel** [comment]
Well, I have just arrived here as a new customer and wonder. Based on my fresh personal experience the app does not yet seem to be ready for prime-time.

**Comment by blacklevel** [comment]
More than a month later - are there any updates / fixes / anything?

**Comment by mjsalam** [comment]
Yup this should be taken down and the seller reported to Gumroad and prohibited from taking any further money from anyone.  It looks extremely bad on gum roads behalf.  It's fraud.
